src/seq_mv/csr_matop.c

```diff
@@ -1910,8 +1910,8 @@ hypre_CSRMatrixComputeRowSum( hypre_CSRMatrix *A,
  * type 0: diag
  *      1: abs diag
  *      2: diag inverse
- *      3: diag inverse sqrt
- *      4: abs diag inverse sqrt
+ *      3: diag inverse hypre_sqrt
```

Comment:
This also should just be sqrt, since it just describes what is done.

Reply:
Thanks for catching these @ulrikeyang. My script was a bit too aggressive :-).